sktime/transformations/series/adi_cv.py

```diff
@@ -47,6 +48,9 @@ class ADICVTransformer(BaseTransformer):
     Default values for the thresholds are taken from the paper by Syntetos/Boylan [1].
     namely, ``adi_threshold = 1.32`` and ``cv2_threshold = 0.49``.
     They can also be adjusted by passing them as parameters to the transformer.
+    Note that Hyndman & Kostenko [2], in their response to Syntetos/Boylan, indicated that
+    there was a more robust threshold for ADI/CV; this can theoretically be accounted for
```

Comment:
the "how" might be useful to tell a reader, if you already cite [2]

Reply:
ok, I just looked, that's a pretty confusing paper. It probably assumes context from multiple references therein.